parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -292,11 +317,39 @@ public <D> FileAppender<D> build() throws IOException {
             .withRowGroupSize(rowGroupSize)
             .withPageSize(pageSize)
             .withDictionaryPageSize(dictionaryPageSize)
-            .build(),
+            // TODO: add .withMaxBloomFilterBytes(bloomFilterMaxBytes) once ParquetWriter.Builder supports it
```

Comment:
If this isn't supported, then we should remove the option to begin with, right?

Reply:
Right. Deleted this comment.